optaplanner-examples/src/main/resources/org/optaplanner/examples/cheaptime/solver/cheapTimeSolverConfig.xml

```diff
@@ -1,7 +1,9 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <solver>
   <!--<environmentMode>FULL_ASSERT</environmentMode>--><!-- To slowly prove there are no bugs in this code -->
-  <!--<moveThreadCount>AUTO</moveThreadCount>--><!-- To solve faster by saturating multiple CPU cores -->
+  <!-- To solve faster by saturating multiple CPU cores. Use value "AUTO" to let OptaPlanner calculate optimal
+       move thread count.-->
+  <moveThreadCount>2</moveThreadCount>
```

Comment:
I am not in favor of this change due to the verbosity. Yes, its more accurate, but now it's 3 lines instead of 1. Let's discuss.

Reply:
Actually, I think that switching to 2 threads by default is not very user-friendly. AUTO is much better default value. I will put there "AUTO" and override to "2" in tests setup - to have it consistent across testing machines.